### PR TITLE
importinto: add more logs to external backend, disable parallel sort

### PR DESCRIPTION
**Pull request by D3Hunter**
### What problem does this PR solve?

Issue Number: ref #47572

Problem Summary:

### What is changed and how it works?

add more logs to external backend, disable parallel sort to speed up encode&sort

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)

with a case of 150g, import takes 58m after vs 70m before, mainly speed up encode&sort step. and CPU curve became more smooth.
after
image
before
image

flush log looks like this:
```
[2023/10/16 16:27:57.768 +00:00] [INFO] [writer.go:445] ["flush kv"] [bytes=268339017] [kv-cnt=18160] [stat-size=42504] [sort-time=250.47596ms] [write-time=1.781394854s] [sort-speed(kv/s)=72.5kB] [write-speed(bytes/s)=143.7MiB] [writerID=data/56625710-9837-4c0e-a3f1-b35490f9a008]
[2023/10/16 16:27:57.963 +00:00] [INFO] [writer.go:445] ["flush kv"] [bytes=268339452] [kv-cnt=18160] [stat-size=42504] [sort-time=53.950553ms] [write-time=1.635531465s] [sort-speed(kv/s)=336.6kB] [write-speed(bytes/s)=156.5MiB] [writerID=data/22a1c4ea-b6aa-4a39-9896-b4a461578107]
```

- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  >   

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by tangenta — APPROVED** [review]

File: br/pkg/lightning/backend/external/writer.go [hunk 1]
```diff
@@ -474,7 +505,6 @@ func (w *Writer) flushKVs(ctx context.Context, fromClose bool) (err error) {
 	w.writeBatch = w.writeBatch[:0]
 	w.rc.reset()
 	w.kvBuffer.Reset()
-	savedBytes = w.batchSize
```
Comment (D3Hunter) [inline br/pkg/lightning/backend/external/writer.go:477]: moved to above

**Review by ywqzzy — APPROVED** [review]

**Comment by D3Hunter** [comment]
will cp to 7.5 when https://github.com/pingcap/tidb/pull/47485 cp, else there might be conflict

**Pull request merged**